contracts/user-contracts/src/main/java/org/java/mentorship/contracts/user/client/UserFeignClient.java

```diff
@@ -0,0 +1,19 @@
+package org.java.mentorship.contracts.user.client;
+
+import org.java.mentorship.contracts.user.dto.User;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import java.util.List;
+
+@FeignClient(value = "userClient",
+        url = "${service.user.url}")
+public interface UserFeignClient {
```

Comment:
This generates a HTTP client that calls the user service.
We should have a client like this for each service and each method so we can make intra-service calls